frontend/src/locales/en.json

```diff
@@ -288,7 +288,39 @@
     "couldNotForge": "Could not forge sword: Insuffucient funds or transaction was denied.",
     "couldNotReforge": "Could not reforge sword: Insufficient funds, Dust, or transaction was denied.",
     "couldNotBurn": "Could not burn sword: Insufficient funds or transaction was denied.",
-    "useMaxSoulAmount": "Use max"
+    "useMaxSoulAmount": "Use max",
+    "specialEvent": "Special Weapon Event",
+    "specialForge": "Special Forge",
+    "specialForgeTooltip": "Forge limited edition weapons created in partnership with other projects.",
+    "noActiveEvents": "There are no active events at this time.",
+    "pickSpecialEvent": "Pick an event to accumulate shards for",
+    "shards": "Shards",
+    "for": "for",
```

Comment:
I'm not sure where it's used, but it's a better practice to just pass a number to the translation instead of cutting it, as it may differ in other languages

Reply:
It's not used in a sentence, just in dropdown options e.g. 3-5* `for` 10 Shards.